### ISSUE TITLE
[new docs site] Add `::: correct` and `::: incorrect` in all rule docs

### ISSUE THREAD

**Issue by mdjermanovic** [eslint/eslint#16013]
In all rule docs, we should wrap correct examples in `::: correct`, and incorrect examples in `::: incorrect` containers.

This adds ✓ or ❌ to the upper right corner.

An example is `semi` rule, updated in https://github.com/eslint/eslint/pull/15998.

https://new.eslint.org/docs/head/rules/semi

![image](https://user-images.githubusercontent.com/44349756/174196547-8c9ea12c-4b15-49af-87c7-73ee946d2eb0.png)

**Comment by DeepshikaS** [comment]
I would like to work on this. I will send a pr.

**Comment by petar8903** [comment]
Start mi bonus

На пт, 17.06.2022 г., 3:02 ч. Milos Djermanovic ***@***.***>
написа:

**Comment by harish-sethuraman** [comment]
> I would like to work on this. I will send a pr.

There are lot of rules. Would you like to split effort?

**Comment by DeepshikaS** [comment]
> > I would like to work on this. I will send a pr.
> 
> There are lot of rules. Would you like to split effort? I can help as well

Hey, I've missed this comment. I had already started working on this pr-https://github.com/eslint/eslint/issues/16013#issuecomment-1159132012 and didn't think it was that complex even if done manually. Initially I had started doing this manually, but while working on the add version pr-https://github.com/eslint/eslint/pull/16022 I found out that this can also be done via script and did using that.
